config/initializers/training_routes.rb

```diff
@@ -627,3 +627,32 @@
     ],
   ),
 ].freeze
+
+POSTGRAD_FUNDED = "Postgrad (fee funded)"
+POSTGRAD_SALARIED = "Postgrad (salaried)"
+UNDERGRAD_FUNDED = "Undergrad (fee funded)"
+OTHER = "Other"
+
+TRAINING_ROUTE_TYPES = {
+  POSTGRAD_FUNDED => [
+    TRAINING_ROUTE_ENUMS[:provider_led_postgrad],
+    TRAINING_ROUTE_ENUMS[:early_years_postgrad],
+  ],
+  POSTGRAD_SALARIED => [
+    TRAINING_ROUTE_ENUMS[:school_direct_salaried],
```

Comment:
This is the one we re-purposing as 

`provider_led_postgrad_salaried` rather then a brand new route right?

Reply:
Yes good spot, I've updated it 👍